src/app/services/Location.ts

```diff
@@ -0,0 +1,42 @@
+import { ISensor } from '../../sensors/Sensor';
+import { AppMessage } from '../appMessage';
+import { SendMessage } from '../../nrfDevice';
+import { Service } from './Service';
+
+const APPID = ['MCELL', 'SCELL', 'WIFI'];
```

Comment:
`CELL_POS` is listed in `src/app/appMessage.ts` enum `appId`, why not here?

Reply:
I will remove it from `appMessage.ts`. 

The BE sends `MCELL` and `SCELL` with appId `CELL_POS` with live data but sends it as `MCELL` and `SCELL` with the location history table, so the FE supports both. I was thinking about a way to make this tool use the `CELL_POS` appId, but it adds more logic then I thought was necessary.